build-farm/make-adopt-build-farm.sh

```diff
@@ -28,18 +28,32 @@ JAVA_FEATURE_VERSION=$(echo "${JAVA_TO_BUILD}" | tr -d "[:alpha:]")
 
 if [ -z "${JAVA_FEATURE_VERSION}" ]
 then
-  # Use Adopt API to get the JDK Head number
-  echo "This appears to be JDK Head. Querying the Adopt API to get the JDK HEAD Number (https://api.adoptopenjdk.net/v3/info/available_releases)..."
-  JAVA_FEATURE_VERSION=$(curl -v https://api.adoptopenjdk.net/v3/info/available_releases | awk '/tip_version/{print$2}')
-    
-  # Checks the api request was successfull and the return value is a number
-  if [ -z "${JAVA_FEATURE_VERSION}" ] || ! [[ "${JAVA_FEATURE_VERSION}" -gt 0 ]]
-  then
-    echo "Failed to query or parse the adopt api. Dumping headers via curl -v https://api.adoptopenjdk.net/v3/info/available_releases..."
-    curl -v https://api.adoptopenjdk.net/v3/info/available_releases
-    exit 1
-  fi
-  echo "JAVA_FEATURE_VERSION is ${JAVA_FEATURE_VERSION}"
+    retryCount=1
+    retryMax=5
+    until [ "$retryCount" -ge "$retryMax" ]
+    do
+        # Use Adopt API to get the JDK Head number
+        echo "This appears to be JDK Head. Querying the Adopt API to get the JDK HEAD Number (https://api.adoptopenjdk.net/v3/info/available_releases)..."
+        JAVA_FEATURE_VERSION=$(curl -v https://api.adoptopenjdk.net/v3/info/available_releases | awk '/tip_version/{print$2}')
+        
+        # Checks the api request was successfull and the return value is a number
+        if [ -z "${JAVA_FEATURE_VERSION}" ] || ! [[ "${JAVA_FEATURE_VERSION}" -gt 0 ]]
+        then
+            echo "Query ${retryCount} failed. Retrying in 30 seconds (max retries = ${retryMax})..."
```

Comment:
Can you add `RETRYWARNING:` to the start of this `echo` statemnet for consistence with what we've done elsewhere to make it easy to scan the logs for occurrences?

Reply:
done